milestone3/backend/index.js

```diff
@@ -0,0 +1,88 @@
+const express = require("express");
+const app = express();
+app.use(express.json());
+const mongoose = require("mongoose");
+const connection_url =
+  "mongodb+srv://rizenson:h4ipassword@cluster0.2ul0y.mongodb.net/RecipesDB?retryWrites=true&w=majority";
+
+mongoose
+  .connect(connection_url)
+  .then(() => console.log("successful connection"))
+  .catch((error) => console.error("connection unsuccessful"));
+
+app.get("/", (req, res) => {
+  res.send("Hello world!");
+});
+
+const RecipeSchema = new mongoose.Schema(
+  {
+    name: String,
+    description: String,
+    imageSrc: String,
+    ingredientList: String,
+    instructionList: String,
+    link: String,
+    courtesyOf: String,
+  },
+  { collection: "Recipes" }
+);
+
+const Recipe = mongoose.model("Recipes", RecipeSchema);
+
+app.post("/api/recipe", async (req, res) => {
+  const {
+    name,
+    description,
+    imageSrc,
+    ingredientList,
+    instructionList,
+    link,
+    courtesyOf,
+  } = req.body;
+  let person = new Recipe({
+    name: name,
```

Comment:
Remember if the attribute and value are the same, you can use shorthand notation to clean up your code a little (although as i'm typing I think I remember seeing that you used a code formatter, and it might prefer this way!)